UX Data Broker Transform/Fetch SysProperty/UIB - Fetch System Property

```diff
@@ -0,0 +1,21 @@
+//Once this is saved, it will show up in UI Builder as a Data Resource which can be selected if the user wants to fetch system property table values.
```

Comment:
@TRIGGEREDcoder please check the [UIB - Fetch System Property]( https://github.com/ServiceNowDevProgram/code-snippets/pull/1478/files#diff-eb6aff1d58c0a2cf92e150998d9bf0c9d12f04203214a31a061be6a4e711c825 ) and update to the correct file format.